test/gtest/uct/uct_p2p_test.cc

```diff
@@ -182,6 +182,13 @@ void uct_p2p_test::test_xfer_multi_mem_type(send_func_t send, size_t min_length,
     /* Trim at 4.1 GB */
     max_length = ucs_min(max_length, (size_t)(4.1 * (double)UCS_GBYTE));
 
+    /* Trim by BAR1 size if relevant */
+    if (mem_type == UCS_MEMORY_TYPE_CUDA ||
+        mem_type == UCS_MEMORY_TYPE_CUDA_MANAGED) {
```

Comment:
@shamisp On most systems, we can't register cuda_managed memory, so maybe we should be returning 0 for get_bar1_free_size() for managed memory?

Reply:
pls use ( ) around inner == expressions